eng/Version.Details.xml

```diff
@@ -9,6 +9,10 @@
       <Uri>https://github.com/microsoft/clrmd</Uri>
       <Sha>d724947392626b66e39b525998a8817447d50380</Sha>
     </Dependency>
+    <Dependency Name="Xamarin.Android.Tools.AndroidSdk" Version="1.0.105-preview.225">
+      <Uri>https://github.com/dotnet/android</Uri>
+      <Sha>bf304cf475308547a2a1a342bb9f2e6825cc33cd</Sha>
+    </Dependency>
```

Comment:
This is probably always going to be "preview" branded and marked as a non-shipping/transport package.

We could track with a subscription, but you probably don't need to update this frequently.

I can manage updating it manually when it matters, such as: `> darc update-dependencies --id 276447`